examples/configs/nodes/dbt/source/stripe/payments.yaml

```diff
@@ -0,0 +1,23 @@
+description: Payments fact table.
+type: source
+columns:
+  id:
+    type: INT
+  orderid:
+    type: INT
+  paymentmethod:
+    type: STR
+  status:
+    type: STR
+  amount:
+    type: INT
+  created:
+    type: DATE
+  _batched_at:
+    type: DATETIME
+tables:
+  postgres:
+  - catalog: null
+    schema: stripe
+    table: payments
+    cost: 10.0
```

Comment:
Maybe we could remove `cost` when there's just a single table? Since it's inherited from the database cost?